internal/lint/cronjob_test.go

```diff
@@ -59,3 +60,54 @@ func TestCronJobLint(t *testing.T) {
 	assert.Equal(t, `[POP-106] No resources requests/limits defined`, ii[5].Message)
 	assert.Equal(t, rules.WarnLevel, ii[5].Level)
 }
+
+func TestCjSpecFor(t *testing.T) {
```

Comment:
These tests really test the same call. Better would be to have tests that actually show the resolved fix i.e no spinach vs with spinach and show container codes exclusions.